### ISSUE TITLE
Grappling

### ISSUE THREAD

**Issue by prosfygas33** [unitedworldwrestling/arena-public#165]
**Arena -scoreboard**
[eg. Arena, Scoreboard, LGTV, etc]

Dear @csabavirag,

I am seeking guidance regarding the creation of a new grappling event, particularly in regard to its competitive format. As a newcomer to this domain, I am keen to discern whether the competition format differs significantly from that of wrestling(Tournament type,ranking type or uww ranking type ). Your insights on this matter would be greatly appreciated.

**Comment by csabavirag** [comment]
Hello @prosfygas33,
There is nothing specific to Grappling in terms of event configuration.

The difference between Olympic Wrestling and Grappling rules is managed via the weight categories you add to your event. So Arena will know what type of bracket to render for Grappling style and what rules to follow if those are different from Olympic styles. 

You just have to add the necessary weight categories to the event either via the UI by start typing "gp" or "gi" to the weight category search widget;
![image](https://github.com/unitedworldwrestling/arena-public/assets/10976654/30dab72f-6d2b-401b-9e49-33dcc9ed8945)
or via the Athlete Import spreadsheet (you must use the appropriate weight category identifier [`gp_m`, `gp_w`, `gp_gi_m`, `gp_gi_w`] under the `Discipline` column. These can be found under the Settings->Sports Config -> Sports section as `identifier`).

Once your athletes are added and draw is done, you can generate matches as you normally do and can start the event.

**Comment by prosfygas33** [comment]
Thank you for your prompt answer.

**Comment by csabavirag** [comment]
No, there is a single onvenue page in Arena to display Start List. Would be nice to have more details on the differences to tell you what the delegate could think about.
